### ISSUE TITLE
Did not work on Mac OS (11.4): ps: unsupported exe keyword

### ISSUE THREAD

**Issue by shagohead** [franciscolourenco/done#109]
On Mac OS 11.4 (don't know about ps in previous Mac OS releases) `exe` keyword not supported for `-o` option.

```
ps: exe: keyword not found
ps: no valid keywords; valid keywords:
test: Expected a combining operator like '-a' at index 3
! %cpu %mem acflag acflg args blocked caught comm command cpu cputime etime f flags gid group ignored inblk inblock jobc ktrace ktracep lim login logname lstart majflt minflt msgrcv msgsnd ni nice nivcsw nsignals nsigs nswap nvcsw nwchan oublk oublock p_ru paddr pagein pcpu pending pgid pid pmem ppid pri pstime putime re rgid rgroup rss ruid ruser sess sig sigmask sl start stat state stime svgid svuid tdev time tpgid tsess tsiz tt tty ucomm uid upr user usrpri utime vsize vsz wchan wq wqb wql wqr xstat = tmux
                                                                                                                                                                                                                                       ^
~/.config/fish/conf.d/done.fish (line 115):
            and test ! (basename (ps -o exe= -p $tmux_fish_ppid)) = "tmux"
                ^
in function '__done_is_tmux_window_active'
        called on line 146 of file ~/.config/fish/conf.d/done.fish
in function '__done_is_process_window_focused'
        called on line 206 of file ~/.config/fish/conf.d/done.fish
in function '__done_ended'
in event handler: handler for generic event 'fish_prompt'
```

**Comment by aystroganov** [comment]
Similar issue is happening on Pop!_OS 20.04, when I uend an ssh session while in tmux:
```
Connection to XXX closed.
error: unknown user-defined format specifier "exe"

Usage:
 ps [options]

 Try 'ps --help <simple|list|output|threads|misc|all>'
  or 'ps --help <s|l|o|t|m|a>'
 for additional help text.

For more details see ps(1).
basename: missing operand
Try 'basename --help' for more information.
```

**Comment by aystroganov** [comment]
I just looked up the format specifiers for my version of `ps`, and it's also missing `exe`, however there are `args` and `comm` (see https://linuxcommand.org/lc3_man_pages/ps1.html), not sure which one it should be.

**Comment by jaideeprana** [comment]
Hey,

Any update on this?
I am also encountering the same issue.

**Comment by hisnawi** [comment]
Is Sam's fix going to be incorporated in this repo? Is there an ETA for the new release with the fix?

**Comment by franciscolourenco** [comment]
Fixed in v1.16.3. Sorry for the delay
